### PR TITLE
CPP-344 Migrated x-dash's script to deploy to Github Pages to this repo

### PR DESCRIPTION
**Pull request by emortong**
Ticket [CPP-344 Publish a script to allow any component to publish a Storybook](https://financialtimes.atlassian.net/browse/CPP-344)
 
 As part of the work for creating a single storybook composed of X-dash's, PageKit's, and other storybooks ([CPP-345](https://financialtimes.atlassian.net/browse/CPP-345)) the tooling is being abstracted from single Storybook repos, so that it can be reused.

This script will be used in the following way during the deploy task:
```
- run:
     name: shared-helper / publish-gh-pages
     command: .circleci/shared-helpers/helper-publish-gh-pages
```

### CODE REVIEW

File: helper-publish-gh-pages [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```
Comment (andygout) [inline helper-publish-gh-pages:1]: Can I request changing the filename from `helper-publish-gh-pages` to `helper-publish-github-pages`? For the sake of a few characters, avoiding abbreviations can make names more immediately understandable for other users.

This was expressed as a [style rule](https://github.com/Financial-Times/dotcom-page-kit/blob/master/contribution.md#general-style) for Page Kit and I think it's a pretty good rule to apply generally.

File: helper-publish-gh-pages [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# HELPER PURPOSE
+#
+# Deploy to Github Pages
+
+
+TARGET_DIR=web/public/*
+TARGET_BRANCH=gh-pages
```
Comment (andygout) [inline helper-publish-gh-pages:9]:
```suggestion
TARGET_BRANCH=github-pages
```

File: secret-squirrel.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ module.exports = {
 			'helper-npm-update',
 			'helper-check-service-ready',
 			'helper-npm-version-and-publish-public',
+			'helper-publish-gh-pages',
```
Comment (andygout) [inline secret-squirrel.js:13]:
```suggestion
			'helper-publish-github-pages',
```

**Review by andygout — COMMENTED** [review]
Nice 👍

Did you use any guides to add these changes that could be added to the PR description for future reference?

**Review by andygout — APPROVED** [review]

**Comment by emortong** [comment]
> Did you use any guides to add these changes that could be added to the PR description for future reference?

@andygout  I followed the [General guidelines for helper scripts](https://github.com/Financial-Times/next-ci-shared-helpers#general-guidelines-for-helper-scripts) that's on the README

**Pull request merged**